bin/bossplot

```diff
@@ -72,6 +97,14 @@ def main():
         help = 'Show the subtracted sky flux instead of the object flux.')
     parser.add_argument('--add-sky', action = 'store_true',
         help = 'Add the subtracted sky to the object flux (overrides show-sky).')
+    parser.add_argument('--flux-range', type=str, default='0.5%:99.5%', metavar="MIN:MAX",
+        help='Limit flux axis range. MIN or MAX may be either a number, percent, or blank.')
+    parser.add_argument('--wlen-range', type=str, default=None, metavar="MIN:MAX",
+        help='Limit wavelength axis range. MIN or MAX may be either a number, percent, or blank.')
+    parser.add_argument('--wdisp-range', type=str, default=None, metavar="MIN:MAX",
+        help='Limit dispersion axis range. MIN or MAX may be either a number, percent, or blank.')
+    parser.add_argument('--figsize', type=str, default='12,8', metavar="WIDTH,HEIGHT",
+        help='Figure dimensions in inches.')
```

Comment:
I guess the DPI has to be set correctly for the values to be real inches.  Does this actually work out?